docs/labs/citizen-integrator-track/lab01/README.adoc

```diff
@@ -0,0 +1,393 @@
+:imagesdir: https://github.com/RedHatWorkshops/dayinthelife-integration/raw/master/docs/labs/citizen-integrator-track/lab01/
```

Comment:
@david-martin these markdown files converted nicely:
- using kramdown
- using docs/edits.sh script for tidy up
- this is the only one I've redirected source for images, you can see pattern and it reveals an issue with web app